checker/tests/index/MinLenFromPositive.java

```diff
@@ -0,0 +1,21 @@
+import org.checkerframework.checker.index.qual.Positive;
+import org.checkerframework.common.value.qual.*;
+
+class MinLenFromPositive {
+
+    void test(@Positive int x) {
+        int @MinLen(1) [] y = new int[x];
+        @IntRange(from = 1) int z = x;
+        @Positive int q = x;
+    }
+
+    @SuppressWarnings("index")
```

Comment:
Please add a subclass of CheckerFrameworkPerDirectoryTest that runs only the Value Checker, but can use `@Postive`.  (I think you just have to put the test class in the checker test directory.)  That way you can make sure that the Value Checker isn't issuing a warning when the LHS is `@Postive`.